_projects/antlr-v4.md

```diff
@@ -14,6 +14,7 @@ mentors:
   - corona10
   - aptrishu
 name: "Integrate ANTLR v4 into coala"
+report: "http://projects.coala.io/GSoC/2018/StatusReport/virresh.html"
```

Comment:
Can you use relative link?

Reply:
I will try that and if possible I will update that change.

Reply:
@li-boxuan I have changed the reports link. But couldn't make it a fine relative link because the `href` was also containing the other parameters like `year`.
Could please tell me if the changes are correct or not ?